The error with GHC 8.2/4 is now:
```
src/Distribution/Simple/I18N/GetText.hs:216:48: error:
    Ambiguous occurrence ‘matchFileGlob’
    It could refer to either ‘Distribution.Simple.Utils.matchFileGlob’,
                             imported from ‘Distribution.Simple.Utils’ at src/Distribution/Simple/I18N/GetText.hs:81:1-42
                          or ‘Internal.matchFileGlob’,
                             imported from ‘Internal’ at src/Distribution/Simple/I18N/GetText.hs:93:71-83
                             (and originally defined at src/Internal.hs:26:1-13)
```
https://github.com/haskell-hvr/hgettext/runs/8209769966?check_suite_focus=true#step:18:31

With GHC 7.4 - 8.0 it is:
```
src/Internal.hs:11:18:
    Could not find module `Distribution.Types.PackageDescription'
```
https://github.com/haskell-hvr/hgettext/runs/8209771066?check_suite_focus=true#step:18:29

--- a/src/Distribution/Simple/I18N/GetText.hs
+++ b/src/Distribution/Simple/I18N/GetText.hs
@@ -78,7 +78,7 @@ import           Distribution.Simple
 import           Distribution.Simple.InstallDirs    as I
 import           Distribution.Simple.LocalBuildInfo
 import           Distribution.Simple.Setup
-import           Distribution.Simple.Utils
+import           Distribution.Simple.Utils          (warn)
 import           Distribution.Verbosity
 
 import           Control.Arrow                      (second)

--- a/src/Internal.hs
+++ b/src/Internal.hs
@@ -5,10 +5,11 @@ module Internal where
 import           Distribution.Simple
 #if MIN_VERSION_Cabal(2,4,0)
 import           Distribution.Simple.Glob    (matchDirFileGlob)
+import           Distribution.Types.PackageDescription
 #else
 import qualified Distribution.Simple.Utils   as Utils (matchFileGlob)
+import           Distribution.PackageDescription
 #endif
-import           Distribution.Types.PackageDescription
 import           Distribution.Verbosity      (Verbosity)
 
 fromPackageName :: PackageName -> String

